src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -238,6 +249,15 @@ internal bool TriggerAppliesToProjection(CompletionContext context, RazorLanguag
                 return true;
             }
 
+            if (context.TriggerCharacter == "(")
```

Comment:
So it essentially "restarts" looking for completions after the `(` and thus the `(` isn't removed when the completion is accepted?

Reply:
Exactly. Restarting completion basically updates the state that VS maintains which it uses to decide what to replace when accepting a completion.

Reply:
That's awesome, thanks!